inst/rmd/qc_report.rmd

```diff
@@ -0,0 +1,32 @@
+---
+params:
+  sample: Example
+  sce: !r SingleCellExperiment::SingleCellExperiment()
+  date: !r Sys.Date()
+
+title: "`r glue::glue('ScPCA QC report for {params$sample}')`"
+author: "CCDL"
+date: "`r params$date`"
```

Comment:
Why not `Sys.Date()` here? I can see the argument for what you've implemented but just wondering what you are potentially future-proofing for...

Reply:
I think that was me playing around a bit. Also solving for the case when we want to forge report dates.